web/app/themes/lawcom/lib/utils.php

```diff
@@ -27,3 +27,9 @@ function project_has_documents($projectId) {
 
   return $documents->have_posts();
 }
+
+// Tell WordPress to use searchform.php from the templates/ directory
+function get_feedback_banner() {
+  locate_template('/templates/feedback-banner.php', true, false);
+}
+add_filter('get_feedback_banner', 'get_feedback_banner');
```

Comment:
Just curious why you made this a function? Also I dont think you need the add_filter.

Reply:
Good question! I was unsure about this bit - and if `utils.php` was the right place for it! What do you think? 

I copied the above code which pulls in the form - is there another way of doing it :D?

Reply:
I would just use get_template_part where your calling get_feedback_banner

Reply:
Ok! I'll switch to that :D is it alright in `utils.php`?

Reply:
I've removed it from `utils` and put it straight into the `header.php` file instead :D